app/src/main/java/org/stepic/droid/ui/activities/MainFeedActivity.kt

```diff
@@ -367,4 +393,16 @@ class MainFeedActivity : BackToExitActivityWithSmartLockBase(),
         val intervalCode = data.getIntExtra(TimeIntervalPickerDialogFragment.resultIntervalCodeKey, TimeIntervalUtil.defaultTimeCode)
         streakPresenter.setStreakTime(intervalCode) // we do not need attach this view, because we need only set in model
     }
+
+    override fun hideBadge() {
+        navigationView.setNotification("", navigationAdapter.getPositionByMenuId(R.id.notifications))
+    }
+
+    override fun setBadge(count: Int) {
+        navigationView.setNotification(if (count > MAX_NOTIFICATION_BADGE_COUNT) {
```

Comment:
I think we can extract this if statement to the method or the variable, because method looks like overwhelmed.